tests/dev/03_Population_Pediatric_Raltegavir/StudyDesign.csv

```diff
@@ -0,0 +1,8 @@
+Organism|Weight,Organism|Weight,Gender,App1|Drugmass,App2|Drugmass
+kg,kg,,mg,mg
+SOURCE_MIN,SOURCE_MAX,SOURCE_EQUALS,TARGET,TARGET
+20,40,1,2,3
+20,40,2,2.5,3.5
+40,60,1,10,12
+40,60,2,14,16
+40,,,20,20

```

Comment:
Interesting example. Actually it is invalid. Was the last line meant to be e.g. _**60**,,,20,20_ ?

Reply:
It should indeed be 60.

Reply:
Why is it invalid ?

It does not make sense but it is still valid. In that case the the last entry would win right? 20 would be the value for all drug mass